### ISSUE TITLE
unable to convert Cloudformation resources properly 

### ISSUE THREAD

**Issue by seifrajhi** [DontShaveTheYak/cf2tf#166]
hello,

I am trying to convert a CloudFormation template to terraform, but I ran into below issue:

```bash
resource "aws_iot_topic_rule_destination" "event_bridge_nuke_schedule" {
  // CF Property(Name) = "EventBridgeNukeSchedule-${local.stack_name}"
  // CF Property(Description) = "Scheduled Event for running AWS Nuke on the target accounts within the specified regions"
  // CF Property(ScheduleExpression) = "cron(0 7 ? * * *)"
  // CF Property(State) = "ENABLED"
  role_arn = aws_iam_role.event_bridge_nuke_schedule_role.arn
  // CF Property(Targets) = [
  //   {
  //     Arn = aws_ec2_instance_state.nuke_step_function.id
  //     RoleArn = aws_iam_role.event_bridge_nuke_schedule_role.arn
  //     Id = "EventBridgeNukeSchedule"
  //     Input = "{
  //   "InputPayLoad": {
  //     "nuke_dry_run": "${var.aws_nuke_dry_run_flag}",
  //     "nuke_version": "${var.aws_nuke_version}",
  //     "region_list": [
  //       "eu-west-1",
  //       "eu-west-3"
  //     ]
  //   }
  // }"
  //   }
  // ]
}
```

My Cloudformation resources template

```yaml
  EventBridgeNukeSchedule:
    Type: AWS::Events::Rule
    Properties:
      Name: !Sub EventBridgeNukeSchedule-${AWS::StackName}
      Description: Scheduled Event for running AWS Nuke on the target accounts within the specified regions
      ScheduleExpression: cron(0 7 ? * * *)
      State:  ENABLED
      RoleArn: !GetAtt EventBridgeNukeScheduleRole.Arn
      Targets:
      -
        Arn: !Ref NukeStepFunction
        RoleArn: !GetAtt EventBridgeNukeScheduleRole.Arn
        Id: EventBridgeNukeSchedule
        Input:
          !Sub |-
            {
              "InputPayLoad": {
                "nuke_dry_run": "${AWSNukeDryRunFlag}",
                "nuke_version": "${AWSNukeVersion}",
                "region_list": [
                  "eu-west-1",
                  "eu-west-3"
                ]
              }
            }
```

**Comment by shadycuz** [comment]
Thank you for opening this. I need to improve the search capability. `cf2tf` thought `AWS::Events::Rule` in Cloudformation was `aws_iot_topic_rule_destination` in Terraform but it should have been `aws_cloudwatch_event_rule`.

**Comment by shadycuz** [comment]
This has been fixed in #188 and will be released soon.
